### PR TITLE
Release 2.13.6

### PR DESCRIPTION
**Pull request by desertaxle**
[RENDERED](https://github.com/PrefectHQ/prefect/blob/release-2.13.6/RELEASE-NOTES.md)

### Checklist

- [ ] This pull request references any related issue by including "closes `<link to issue>`"
	- If no issue exists and your change is not a small fix, please [create an issue](https://github.com/PrefectHQ/prefect/issues/new/choose) first.
- [ ] This pull request includes tests or only affects documentation.
- [ ] This pull request includes a label categorizing the change e.g. `fix`, `feature`, `enhancement`, `docs`.
  

For documentation changes:

- [ ] This pull request includes redirect settings in `netlify.toml` for files that are removed or renamed

### CODE REVIEW

File: RELEASE-NOTES.md [hunk 1]
```diff
@@ -1,5 +1,37 @@
...
+### New default option for results storage
+
+
+
+See the following pull request for details:
+- https://github.com/PrefectHQ/prefect/pull/10925
+
+### Experimental support for enhanced cancellation
+
+
```
Comment (zzstoatzz) [inline RELEASE-NOTES.md:14]:
```suggestion
# Specify a default result storage block as a setting
Previously, specifying result storage blocks necessitated changes in the @flow / @task decorator. Now, with [#10925](https://github.com/PrefectHQ/prefect/pull/10925), a `PREFECT_DEFAULT_RESULT_STORAGE_BLOCK` setting has been introduced, allowing users to set a default storage block on a work pool or via infra overrides for a deployment. This enhancement enables easier swapping of result storages by just updating the environment in the UI or in your prefect.yaml, eliminating the need to alter your flow source code.
```

File: RELEASE-NOTES.md [hunk 2]
```diff
@@ -1,5 +1,47 @@
...
+### Experimental support for enhanced cancellation
+
+We're introducing a new experimental feature that will enable more consistent and reliable cancellation of flow runs. 
+
+To enable enhanced cancellation, set the 'PREFECT_EXPERIMENTAL_ENABLE_ENHANCED_CANCELLATION' setting on your worker or agents to `True`:
+
+```bash
+prefect config set PREFECT_EXPERIMENTAL_ENABLE_ENHANCED_CANCELLATION=True
+```
+
```
Comment (serinamarie) [inline RELEASE-NOTES.md:33]: Would it be worth adding a small blurb to the [docs](https://docs.prefect.io/2.13.5/concepts/flows/?h=cancel#cancel-a-flow-run)?

File: RELEASE-NOTES.md [hunk 3]
```diff
@@ -1,5 +1,62 @@
...
+When enabled, you can cancel flow runs where cancellation commonly fails, like if your worker is offline. We will continue to develop enhanced cancellation to improve its reliability and performance. If you encounter any issues, please let us know in Slack or with a Github issue.
```
Comment (serinamarie) [inline RELEASE-NOTES.md:34]:
```suggestion
When enabled, you can cancel flow runs where cancellation can fail, such as when your worker is offline. We will continue to develop enhanced cancellation to improve its reliability and performance. If you encounter any issues, please let us know in Slack or with a Github issue.
```

File: RELEASE-NOTES.md [hunk 4]
```diff
@@ -1,5 +1,62 @@
...
+
+Note: if you are using the Kubernetes worker, you will need to update your `prefect-kubernetes` installation to `0.3.1`, and if you are using the Cloud Run or Vertex AI workers, you will need to update your `prefect-gcp` installation to `0.5.1`.
```
Comment (serinamarie) [inline RELEASE-NOTES.md:38]:
```suggestion
Note: If you are using the Kubernetes worker, you will need to update your `prefect-kubernetes` installation to `0.3.1`. If you are using the Cloud Run or Vertex AI workers, you will need to update your `prefect-gcp` installation to `0.5.1`.
```

File: RELEASE-NOTES.md (hunk 2 above)
Comment (desertaxle) [reply RELEASE-NOTES.md:35]: Yeah, I'll open a separate PR for that.

File: RELEASE-NOTES.md [hunk 5]
```diff
@@ -1,5 +1,66 @@
...
+- name: my-super-cool-deployment
+  entrypoint: some_directory/some_file.py:my_flow
+  work_pool:
+    name: ecs-pool
+    schedule:
+      cron: "0 20 * * 1-5"
+    job_variables:
+      env:
+        PREFECT_DEFAULT_RESULT_STORAGE_BLOCK: s3/my-s3-bucket-block-name
+```
```
Comment (zzstoatzz) [inline RELEASE-NOTES.md:19]: whoops
```suggestion
```yaml
- name: my-super-cool-deployment
  entrypoint: some_directory/some_file.py:my_flow
  work_pool:
    schedule:
      cron: "0 20 * * 1-5"
    name: ecs-pool
    job_variables:
      env:
        PREFECT_DEFAULT_RESULT_STORAGE_BLOCK: s3/my-s3-bucket-block-name
```
```

File: RELEASE-NOTES.md [hunk 6]
```diff
@@ -1,5 +1,65 @@
...
+- name: my-super-cool-deployment
+  entrypoint: some_directory/some_file.py:my_flow
+  schedule:
+    cron: "0 20 * * 1-5"
+  work_pool:
+    name: ecs-pool
+    job_variables:
+      env:
+        PREFECT_DEFAULT_RESULT_STORAGE_BLOCK: s3/my-s3-bucket-block-name
+
```
Comment (serinamarie) [inline RELEASE-NOTES.md:19]:
```suggestion
```
```

Comment (zzstoatzz) [inline RELEASE-NOTES.md:19]:
```suggestion
```yaml
- name: my-super-cool-deployment
  entrypoint: some_directory/some_file.py:my_flow
  schedule:
    cron: "0 20 * * 1-5"
  work_pool:
    name: ecs-pool
    job_variables:
      env:
        PREFECT_DEFAULT_RESULT_STORAGE_BLOCK: s3/my-s3-bucket-block-name
```
```

File: RELEASE-NOTES.md (hunk 2 above)
Comment (desertaxle) [reply RELEASE-NOTES.md:34]: PR here: https://github.com/PrefectHQ/prefect/pull/10944

File: RELEASE-NOTES.md [hunk 7]
```diff
@@ -1,5 +1,65 @@
...
+- name: my-super-cool-deployment
+  entrypoint: some_directory/some_file.py:my_flow
+  schedule:
+    cron: "0 20 * * 1-5"
+  work_pool:
+    name: ecs-pool
+    job_variables:
+      env:
+        PREFECT_DEFAULT_RESULT_STORAGE_BLOCK: s3/my-s3-bucket-block-name
+```
```
Comment (serinamarie) [inline RELEASE-NOTES.md:19]: should we specify this as being in the deployment section?

Comment (zzstoatzz) [reply RELEASE-NOTES.md:19]: yep - good catch

File: RELEASE-NOTES.md [hunk 8]
```diff
@@ -1,5 +1,66 @@
...
+This enhancement enables easier swapping of result storages by just updating the environment in the UI or in your `prefect.yaml`, eliminating the need to alter your flow source code.
+
+See the following pull request for details:
+- https://github.com/PrefectHQ/prefect/pull/10925
+
+### Experimental support for enhanced cancellation
+
+We're introducing a new experimental feature that will enable more consistent and reliable cancellation of flow runs. 
+
+To enable enhanced cancellation, set the 'PREFECT_EXPERIMENTAL_ENABLE_ENHANCED_CANCELLATION' setting on your worker or agents to `True`:
```
Comment (serinamarie) [inline RELEASE-NOTES.md:30]:
```suggestion
To enable enhanced cancellation, set the `PREFECT_EXPERIMENTAL_ENABLE_ENHANCED_CANCELLATION` setting on your worker or agents to `True`:
```

File: RELEASE-NOTES.md [hunk 9]
```diff
@@ -1,5 +1,66 @@
...
+- Remove icons from side nav for improved readability — https://github.com/PrefectHQ/prefect/pull/10908
+- Update deployments tutorial for consistent styling — https://github.com/PrefectHQ/prefect/pull/10911
+- Fix typo in CLI command in deployments tutorial — https://github.com/PrefectHQ/prefect/pull/10937
+- Fix typo in logging guide — https://github.com/PrefectHQ/prefect/pull/10936
+- Update documentation styling  — https://github.com/PrefectHQ/prefect/pull/10913
+
+## Contributors
+* @Sun-of-a-beach made their first contribution in https://github.com/PrefectHQ/prefect/pull/10937
+- @manaw
+
```
Comment (serinamarie) [inline RELEASE-NOTES.md:60]:
```suggestion
```

File: RELEASE-NOTES.md [hunk 10]
```diff
@@ -1,5 +1,66 @@
...
+### Documentation
+- Add a guide for using the Prefect client — https://github.com/PrefectHQ/prefect/pull/10924
+- Remove icons from side nav for improved readability — https://github.com/PrefectHQ/prefect/pull/10908
+- Update deployments tutorial for consistent styling — https://github.com/PrefectHQ/prefect/pull/10911
+- Fix typo in CLI command in deployments tutorial — https://github.com/PrefectHQ/prefect/pull/10937
+- Fix typo in logging guide — https://github.com/PrefectHQ/prefect/pull/10936
+- Update documentation styling  — https://github.com/PrefectHQ/prefect/pull/10913
+
+## Contributors
```
Comment (serinamarie) [inline RELEASE-NOTES.md:57]:
```suggestion
### Contributors
```

File: RELEASE-NOTES.md [hunk 11]
```diff
@@ -1,5 +1,66 @@
...
+
+See the following pull requests for details:
+- https://github.com/PrefectHQ/prefect/pull/10920
+- https://github.com/PrefectHQ/prefect/pull/10944
+
+### Enhancements
+- Add link to Prefect Cloud information in the Prefect UI — https://github.com/PrefectHQ/prefect/pull/10909
+
+### Fixes
+- Avoid prompting for remote storage if a global pull step is already defined - https://github.com/PrefectHQ/prefect/pull/10941
```
Comment (serinamarie) [inline RELEASE-NOTES.md:48]:
```suggestion
- Avoid `prefect deploy` prompt for remote storage if a global pull step is already defined - https://github.com/PrefectHQ/prefect/pull/10941
```

**Review by zzstoatzz — APPROVED** [review]

**Review by serinamarie — APPROVED** [review]
LGTM!

**Pull request merged**